runtime/guard-jwt/src/test/java/io/aklivity/zilla/runtime/guard/jwt/internal/config/AbstractJwtConfigAdapterTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.guard.jwt.internal.config;
+
+import jakarta.json.bind.Jsonb;
+import jakarta.json.bind.JsonbBuilder;
+import jakarta.json.bind.JsonbConfig;
+import jakarta.json.bind.adapter.JsonbAdapter;
+
+class AbstractJwtConfigAdapterTest
+{
+    protected Jsonb jsonb;
+
+    protected void initJson(JsonbAdapter<?, ?> adapter)
+    {
+        JsonbConfig config = new JsonbConfig()
+                .withAdapters(adapter);
+        jsonb = JsonbBuilder.create(config);
+    }
+}
```

Comment:
Please remove this file and duplicate the code in each subclass instead.
The design for tests (linearity) is different from the design for implementation (abstraction, reuse).

Reply:
ok, thanks. will do